### PR TITLE
Add some new mispronunciation words found in working.

### PR DESCRIPTION
**Pull request by D50000**
- Add `asynchronous`
- Add `error`

🤣😃👌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 | [archive](https://www.ldoceonline.com/dictionary/archive) [🔊](https://www.ldoceonline.com/media/english/ameProns/laadarchive.mp3) | ✅ ['ɑːkaɪv] | ❌ ['ətʃɪv] |
 | [array](https://www.ldoceonline.com/dictionary/array) [🔊](https://www.ldoceonline.com/media/english/ameProns/array1.mp3) | ✅ [ə'rei] | ❌ [æ'rei] |
 | [aspect](https://www.ldoceonline.com/dictionary/aspect) [🔊](https://www.ldoceonline.com/media/english/ameProns/aspect.mp3) | ✅ ['æspekt] | ❌ [ə'spekt] |
-| [avatar](https://www.ldoceonline.com/dictionary/avatar) [🔊](https://www.ldoceonline.com/media/english/ameProns/avatar.mp3) | ✅ ['ævətɑː] | ❌ [ə'vʌtɑ] |
+| [asynchronous](https://www.ldoceonline.com/dictionary/asynchronous) [🔊](https://www.ldoceonline.com/media/english/ameProns/asynchronous1004a.mp3?version=1.2.68) | ✅ [eɪˈsɪŋkrənəs] | ❌ [əˈsɪŋkrənəs] |
```
Comment (pjchender) [inline README.md]: Might not need the `version`.


```suggestion
| [asynchronous](https://www.ldoceonline.com/dictionary/asynchronous) [🔊](https://www.ldoceonline.com/media/english/ameProns/asynchronous1004a.mp3) | ✅ [eɪˈsɪŋkrənəs] | ❌ [əˈsɪŋkrənəs] |
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@
 | [digest](https://www.ldoceonline.com/dictionary/digest) [🔊](https://www.ldoceonline.com/media/english/ameProns/l3digest.mp3) | ✅ n. ['dɑɪdʒɛst] v. [dɑɪ'dʒɛst] | ❌ ['dɪgɛst] |
 | [Django](https://www.djangoproject.com/start/overview/) [🔊](https://www.google.com/search?q=django+pronunciation) | ✅ [ˈdʒæŋɡoʊ], JANG-goh| ❌ [diˈdʒæŋɡoʊ] |
 | [doc](https://www.ldoceonline.com/dictionary/doc) [🔊](https://www.ldoceonline.com/media/english/ameProns/doc.mp3) | ✅ [dɒk]| ❌ [daʊk] |
+| [error](https://www.ldoceonline.com/dictionary/error) [🔊](https://www.ldoceonline.com/media/english/ameProns/error.mp3?version=1.2.68) | ✅ [ˈerər] | ❌ [ērər] |
```
Comment (pjchender) [inline README.md]: ditto


```suggestion
| [error](https://www.ldoceonline.com/dictionary/error) [🔊](https://www.ldoceonline.com/media/english/ameProns/error.mp3) | ✅ [ˈerər] | ❌ [ērər] |
```

**Review by pjchender — COMMENTED** [review]
Nice. I found I speak the `asynchronous` wrongly before 😅 .

File: README.md (hunk 2 above)
Comment (D50000) [reply README.md]: Oh, that right. 👌

**Review by pjchender — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**